scripts/process.py

```diff
@@ -1,3 +1,9 @@
+'''
+This script will import and clean data from the JHU CSSE COVID-19 dataset.
+
+Dependencies: dataflows==0.0.71 (pip install dataflows)
```

Comment:
The dependencies should not be listed here: this is why `requirements.txt` exists. This leads to duplicate code, additional maintenance cost (need to update twice the same information) and will likely get out of sync once we update dependencies, leading to confusion.

The process is also described here: https://github.com/datasets/covid-19#preparation